src/entity/posts/utils.ts

```diff
@@ -389,3 +391,75 @@ export const updateUsedImagesInContent = async (
     },
   );
 };
+
+export const addRelatedPosts = async ({
+  entityManager,
+  postId,
+  entries,
+  relationType,
+}: {
+  entityManager: EntityManager;
+  postId: Post['id'];
+  entries: string[];
+  relationType: PostRelationType;
+}): Promise<Post[]> => {
+  if (!entries.length) {
+    return [];
+  }
+
+  const posts = await entityManager.getRepository(Post).findBy({
+    yggdrasilId: In(entries),
+  });
+
+  await entityManager
+    .getRepository(PostRelation)
+    .createQueryBuilder()
+    .insert()
+    .values(
+      posts.map((post) => ({
+        postId,
+        relatedPostId: post.id,
```

Comment:
`relatedPostId` is the collection id? Just making sure because I'm always confused by the name